tests/rpc/client_lwt_new.ml

```diff
@@ -125,7 +125,7 @@ let main () =
   Server.rpc4 ImplM.rpc4;
 
   let funcs = Server.implementation in
-  let rpc = rpc (Rpc_lwt.server funcs) in
+  let rpc r = rpc (Rpc_lwt.server funcs |> Obj.magic) r |> Obj.magic in
```

Comment:
Although leaking Obj.magic calls into the code that end-users of the library write is a lot more worrying.

Reply:
I thin this should be avoidable as well. One of these calls is actually `Rpc_lwt.M.lwt` but I realised it only later

Reply:
Anyways I agree with you that, if this requires leaking `Obj.magic`, then is the wrong approach.